quickfixj-core/src/main/java/quickfix/DataDictionary.java

```diff
@@ -561,6 +562,25 @@ public boolean isAllowUnknownMessageFields() {
         return allowUnknownMessageFields;
     }
 
+    public boolean isFirstFieldInGroupIsDelimiter() {
+        return firstFieldInGroupIsDelimiter;
+    }
+    /**
+     * Controls whether any field which is
+     * first in the group would be used as delimiter
+     *
+     * @param flag true = use first field from message, false = follow data dictionary
+     * Must be used with enabled {@link #setCheckUnorderedGroupFields(boolean)}
```

Comment:
I could not understand yet why `setCheckUnorderedGroupFields` has to be enabled for this.

Reply:
Thanks, was an oversight, should be `with disabled`